examples/quickstart.py

```diff
@@ -1,78 +1,21 @@
-"""Trains BC, GAIL and AIRL models on saved CartPole-v1 demonstrations."""
```

Comment:
For me the purpose is to have a minimal example, that fits (more or less, if you ignore the imports) on a single screen.
It also allows you to quickly launch something without having to spin up a jupyter notebook (in case that adds friction).
However, I see your point that the difference to the `train_bc.ipynb` is minimal right now. The only thing I really like is that in the python script we can set render=True to actually show what is going on.

Reply:
Thanks for the explanation, yeah I can see the value of keeping this.